docs/source/example_notebooks/dowhy_functional_api.ipynb

```diff
@@ -159,18 +165,18 @@
     "identified_estimand = identify_effect(graph, treatment_name, outcome_name)\n",
     "\n",
     "# auto_identify_effect example with extra parameters:\n",
-    "identified_estimand_auto = identify_effect_auto(\n",
-    "    graph,\n",
-    "    treatment_name,\n",
-    "    outcome_name,\n",
-    "    estimand_type=EstimandType.NONPARAMETRIC_ATE,\n",
-    "    backdoor_adjustment=BackdoorAdjustment.BACKDOOR_EFFICIENT,\n",
-    ")\n",
+    "# identified_estimand_auto = identify_effect_auto(\n",
```

Comment:
Why comment this stuff out? Should we delete it itstead?

Reply:
Those are other examples of executing the same code above, I commented them to avoid making this notebook take more time executing. A user could just copy them to use the API in a slightly different way. But now that I write this, I realize that this also works as test and making sure that it actually works :D I'll uncomment them again.

Reply:
Just want to note that we shouldn't use random seeds in unit tests. However, here it's a notebook, so its fine :)

Reply:
We use notebooks as unit tests though, so we should probably disable random seeds in all of them by default.